### DISCUSSION TITLE
what does the primary hint do ?

### DISCUSSION THREAD

**Discussion by davecramer** [pgagroal/pgagroal#149] [General]
see $subject

**Comment by jesperpedersen** [comment]
It will select that server as the primary instead of going in order of the definitions.

**Reply by davecramer** [reply]
Hmm.. ok, so you give it a list of servers and how do you tell it you prefer the primary ?

**Reply by jesperpedersen** [reply]
It will prefer a primary - only if none is found will it choose a replica
